src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -95,103 +106,149 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
       setIsPublishing(false);
     }
   };
+  const closeStepper = () => {
+    open();
+  };
+
+  /* Start beforeunload event */
+  // This section triggers browser response to unsaved items when the stepper modal is open/active
```

Comment:
nit: use the standard [JSDoc format](https://jsdoc.app/about-getting-started.html) for comments like this.